slugbuilder/builder/build.sh

```diff
@@ -187,25 +222,13 @@ fi
 
 
 ## Produce slug
-
-if [[ -f "${build_root}/.slugignore" ]]; then
-  tar \
-    --exclude='./.git' \
-    --use-compress-program=pigz \
-    -X "${build_root}/.slugignore" \
-    -C ${build_root} \
-    -cf ${slug_file} \
-    . \
-    | cat
-else
-  tar \
-    --exclude='./.git' \
-    --use-compress-program=pigz \
-    -C ${build_root} \
-    -cf ${slug_file} \
-    . \
-    | cat
-fi
+tar \
+  --exclude='./.git' \
+  --use-compress-program=pigz \
+  -C ${build_root} \
+  -cf ${slug_file} \
+  . \
+  | cat
```

Comment:
Yeah I didn't look into that but I too am confused that it pipes to cat now that I look at it closer.